src/ext.rs

```diff
@@ -87,19 +86,19 @@ mod external {
 	}
 }
 
-/// Halt execution and register account for deletion.
+/// Halt execution and register account for deletion
 ///
 /// Value of the current account will be tranfered to `refund` address.
 pub fn suicide(refund: &Address) -> ! {
 	unsafe { external::suicide(refund.as_ptr()); }
 }
 
-/// Returns balance of the given address.
+/// Get balance of the given account
```

Comment:
What if no such address? Should be explained with explanation from other sources

Reply:
Balance of not existing account is 0. There is no way to check if account is exist or not. I think it will be confusing if we write "If account is not exists it returns 0" - because there is no "definition of existence of accounts", in other words, from contract view all accounts exists. So with this mention we're adding a new invariant which is redundant.

Reply:
Redundant to you, but not for general user